### ISSUE TITLE
Any love for the old light theme?

### ISSUE THREAD

**Issue by Nantris** [blainesch/electron-notifier#26]
I really liked the light theme personally. I will likely create a fork with a more chromelike notification style in time personally, but for people who are not inclined to do something like that it would be helpful maybe.

I think dark vs light really depends on the application you're building. Both are nice.

**Comment by bayleedev** [comment]
@Slapbox I liked the theme as well, but it's not cross platform. If you want we could morph the current theme into a light theme and give the option for light/dark and possibly a "css" option for custom themes.

**Comment by Nantris** [comment]
What about it wasn't cross platform?

By the way, I discovered there is an Electron function, `systemPreferences.isAeroGlassEnabled()` to determine if transparency is supported. I remember transparency causing some issues. This could enable a seamless solution with and without transparency.

**Comment by bayleedev** [comment]
So if transparency is turned on, we could use the old light theme, and if not use the new dark theme? Sounds a little weird to change the entire layout it based on that.

It was a transparency issue, and it looked ugly without transparency so I had my friend create a new notification style that looked good without transparency all together, which happened to be dark.

**Comment by Nantris** [comment]
Certainly not saying we need transparency, I just recall seeing it was causing issues. I was not aware of the function to check for support until recently.

Was the transparency the only thing that made the old design unworkable crossplatform?

**Comment by bayleedev** [comment]
Correct. Before, we made it transparent and added the dropshadow via CSS so it was consistent across platforms. ]:

**Comment by allannienhuis** [comment]
+1 for the idea of applying custom styles - something simple perhaps like pass in a path to a custom stylesheet to link?  I'm about to try customisation by injecting a stylesheet into the window, but it would be really convenient to have that in the api.

Thanks for the great library!
